test/build.info

```diff
@@ -298,7 +298,7 @@ IF[{- !$disabled{tests} -}]
 
   IF[{- $disabled{shared} -}]
     PROGRAMS_NO_INST=wpackettest cipher_overhead_test
-    SOURCE[wpackettest]=wpackettest.c testutil.c
+    SOURCE[wpackettest]=wpackettest.c testutil.c test_main_custom.h
```

Comment:
Because originally wpackettest needed to access internal symbols from libssl which are not exported by shared libraries (i.e. WPACKET_*). They are available in a static build though. With the internal test build changes though we should be able to do this now.